agreementbot/agreementworker.go

```diff
@@ -368,35 +426,30 @@ func (b *BaseAgreementWorker) InitiateNewAgreement(cph ConsumerProtocolHandler,
 
 	// get the node type for later use
 	nodeType := wi.Device.GetNodeType()
-	// get the node max heartbeat interval if it's set on the node
-	nodeMaxHBInterval := exchangeDev.HeartbeatIntv.MaxInterval
-
-	// if the node max heartbeat interval is not set on the node, then get if from the org
-	if nodeMaxHBInterval == 0 {
-		exchOrg, err := exchange.GetOrganization(b.config.Collaborators.HTTPClientFactory, exchange.GetOrg(wi.Device.Id), b.config.AgreementBot.ExchangeURL, cph.GetExchangeId(), cph.GetExchangeToken())
-		if err != nil {
-			glog.Errorf(BAWlogstring(workerId, fmt.Errorf("Unable to get org %v from exchange: %v", exchange.GetOrg(wi.Device.Id), err)))
-		}
-		nodeMaxHBInterval = exchOrg.HeartbeatIntv.MaxInterval
-	}
 
 	// There could be more than 1 workload version in the consumer policy, and each version might NOT require the exact same
 	// services/microservices (and versions), so we first need to choose a workload. Choosing a workload is based on the priority of
 	// each workload and whether or not this workload has been tried before. Also, iterate the loop more than once if we choose
 	// a workload entry that turns out to be unsupportable by the device.
-	msgPrinter := i18n.GetMessagePrinter()
 	foundWorkload := false
 	var workload, lastWorkload *policy.Workload
 	svcIds := []string{} // stores the service ids for all the services, top level and dependent services
 	found := true        // if the service policy can be found from the businesspol_manager
 	var servicePol *externalpolicy.ExternalPolicy
 
+        var wlUsage *persistence.WorkloadUsage = nil
```

Comment:
some kind of a white space/tab thing going on here. If you run make format, it might clean this up for you.

Reply:
done